src/Materials/Textures/babylon.dynamicTexture.ts

```diff
@@ -75,11 +75,15 @@ module BABYLON {
             }
 
             this._context.font = font;
-            if (x === null) {
+            if (x === null  || typeof x === "undefined") {

```

Comment:
yeah, sorry, gotten the habit of using typeof everywhere, but ofc in this case neither x or y will be undeclared.. will update in a minute :)